src/main/java/at/feldim2425/moreoverlays/lightoverlay/LightOverlayRenderer.java

```diff
@@ -18,21 +18,40 @@
 import net.minecraft.world.EnumSkyBlock;
 import net.minecraft.world.World;
 import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.Biome.SpawnListEntry;
+
 import org.apache.commons.lang3.tuple.Pair;
 import org.lwjgl.opengl.GL11;
 
 import java.util.LinkedList;
 import java.util.List;
 
-public class LightOverlayRenderer {
+public abstract class LightOverlayRenderer {
 
-	private final static ResourceLocation BLANK_TEX = new ResourceLocation(MoreOverlays.MOD_ID, "textures/blank.png");
-	private final static AxisAlignedBB TEST_BB = new AxisAlignedBB(0.6D / 2D, 0, 0.6D / 2D, 1D - 0.6D / 2D, 1D, 1D - 0.6D / 2D);
-	private static final List<Pair<BlockPos, Byte>> overlayCache = new LinkedList<>();
-	private static RenderManager render = Minecraft.getMinecraft().getRenderManager();
+	private static LightOverlayRenderer INSTANCE;
+	public static LightOverlayRenderer instance() {
+		if(INSTANCE == null) {
+			try {
+				Class.forName("drzhark.customspawner.CustomSpawner", false, LightOverlayRenderer.class.getClassLoader());
+				INSTANCE = new LightOverlayRendererZharkCustomSpawner();
+			}
+			catch (ClassNotFoundException e) {
+				INSTANCE = new LightOverlayRendererVanilla();
+			}
+		}
+			
+		return INSTANCE;
+	}
```

Comment:
I have to think about this method of replacing the renderer. Maybe I will move all of the mod handling into an extra package. Since it seems like I will have to add support for more mods or add an API for these implementations.